x/incentive/keeper/store.go

```diff
@@ -116,9 +112,8 @@ func (k Keeper) setIncentiveProgram(
 		return sdkerrors.ErrInvalidRequest.Wrapf("program %d already exists with status %d", program.ID, s)
 	}
 
-	kvStore := k.KVStore(ctx)
 	key := keyIncentiveProgram(program.ID, status)
-	return store.SetObject(kvStore, k.cdc, key, &program, "incentive program")
+	return k.setObject(ctx, key, &program, "incentive program")
```

Comment:
same concern - we have some incentive programs in state already, so does this require a migration?